src/content/main-1-x-x.ts

```diff
@@ -21,7 +21,7 @@ function createWindow () {
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 // Some APIs can only be used after this event occurs.
-app.on('ready', createWindow)
+app.whenReady().then(createWindow)
```

Comment:
According to https://github.com/electron/electron/commit/fcc82ebd353d1704e111d4d84a8a1e5675840dff `whenReady` was added in v3, so we shouldn't change it here.

```suggestion
app.on('ready', createWindow)
```

Reply:
Hmm... Good catch. If we're supporting Electron 2 in Fiddle, we would have to hack together another `src/content/main-2-x-x.ts` (not ideal).